Derpitcraft/README.md

```diff
@@ -78,13 +78,13 @@ All the following rules can be summarised simply, in one line. "Be kind to other
 
     Derpitcraft recommends using one of two **optional Fabric modpacks**. These enhance client-side performance. They are available at https://tinyurl.com/ds9modpack. Both of these packs include *simple voice chat*. There is a lite version and a normal verison. The lite version is considerably smaller.
 
-    **Proximity Chat System**: Derpitcraft has a proximity chat system powered by the *simple voice chat* mod. Download this mod from https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat to play with Proximity Chat on Derpitcraft. It is optional. To read more on how to use this system, read the guide on it below.
+    **Proximity Chat System**: Derpitcraft has a proximity chat system powered by the *Simple Voice Chat* mod. Download this mod from https://www.curseforge.com/minecraft/mc-mods/simple-voice-chat to play with Proximity Chat on Derpitcraft. It is optional. To read more on how to use this system, read the guide on it below.
 
-10. **IGN Policy**: We **DON'T** support changing your IGN mid-season. We are a **cracked server**, and you will **lose your items and player details** if you change your IGN. Do not change your IGN once you start playing with a specific IGN. *Raise a ticket if this is absolutely necessary.*
+10. **IGN Policy**: We **DON'T** support changing your IGN mid-season. We are a **cracked server**, and you will **lose your items and player details** if you change your IGN. Do not change your IGN once you start playing with a specific IGN. **Raise a ticket if this is absolutely necessary.**
 
 11.  **Lag Policy**: If you are experiencing lag, please wait a while for it to pass. If it stays and worsens, ping a moderator after **3 minutes** of continuous lag. Keep the following in your mind.
 
-     - If you think you have died due to a lag spike/ TPS drop, we will require **videographic evidence** to confirm it and restore your items. Without video, we cannot restore your items.  
+     - If you think you have died due to a lag spike, TPS drop(under 5 TPS), or bad ping(more than 1000 ms), we will require **videographic evidence** to confirm it and restore your items. Without video, we cannot restore your items.  
```

Comment:
"
     - If you think you have died due to a lag spike, TPS drop (under 5 TPS), or bad ping (more than 1000ms), we will require **videographic evidence** to confirm it and restore your items. Without video, we cannot restore your items.
"

Reply:
?

Reply:
edit it with the changed text. changed spacing

Reply:
```suggestion
    - If you think you have died due to a lag spike, TPS drop (under 5 TPS), or bad ping (more than 1000ms), we will require videographic evidence to confirm it and restore your items. Without video, we cannot restore your items. 
```